neurobooth_os/log_manager.py

```diff
@@ -49,9 +47,12 @@ def make_session_logger_debug(
 
 
 def make_default_logger(
-        log_path=DEFAULT_LOG_PATH,
-        log_level=logging.DEBUG,
+        log_path: Optional[str] = None,
```

Comment:
This is mostly going away soon, otherwise I would suggest comments that make it clear that the method doesn't really default to None for the log path.